### ISSUE TITLE
Could not download files

### ISSUE THREAD

**Issue by tampe125** [github/git-lfs#836]
I'm having some troubles retrieving the files I stored using git-lfs.
On a computer I correctly setup GIT LFS to track and store the large files, now I'm on another pc and I want to actually fetch those files.
At the moment I only have the pointers to such files, so I tried to use `git lfs fetch` to download those files.
The download simply hangs, nothing happens (they are just 50Mb of stuff, so the download should be quite fast).
So I tried with `git checkout .`, no joy, download is still hanging.
Any suggestions? FWIW I'm connecting to the repo using HTTPS

**Comment by technoweenie** [comment]
What does `git lfs env` show?  What about `GIT_TRACE=1 git lfs fetch`?

**Comment by tampe125** [comment]
**git lfs env**
```
git-lfs/1.0.2 (GitHub; windows amd64; go 1.5.1; git 0566698)
git version 2.5.3

Endpoint=https://tampe125@github.com/akeeba/backup-core.git/info/lfs (auth=basic)
LocalWorkingDir=C:\cygdrive\c\Users\tampe125\Documents\git\backup
LocalGitDir=C:\Users\tampe125\Documents\git\backup\.git
LocalGitStorageDir=C:\Users\tampe125\Documents\git\backup\.git
LocalMediaDir=C:\Users\tampe125\Documents\git\backup\.git\lfs\objects
TempDir=C:\Users\tampe125\Documents\git\backup\.git\lfs\tmp
ConcurrentTransfers=3
BatchTransfer=true
git config filter.lfs.smudge = "git-lfs smudge %f"
git config filter.lfs.clean = "git-lfs clean %f"
```

**GIT_TRACE=1 git lfs fetch**
```
08:19:19.928112 git.c:558               trace: exec: 'git-lfs' 'fetch'
08:19:19.931153 run-command.c:335       trace: run_command: 'git-lfs' 'fetch'
trace git-lfs: run_command: 'git' rev-parse --abbrev-ref HEAD
trace git-lfs: run_command: 'git' config branch.feature/MYBRANCH.remote
trace git-lfs: run_command: 'git' rev-parse HEAD --symbolic-full-name HEAD
trace git-lfs: run_command: 'git' config -l
trace git-lfs: run_command: 'git' config -l -f C:\Users\tampe125\Documents\git\MYREPO\.git\config
Fetching MYBRANCH
trace git-lfs: run_command: git rev-list --objects --no-walk a1700efd8669eef04c1776b2111cf79172a68e81
trace git-lfs: run_command: git cat-file --batch-check
trace git-lfs: run_command: git cat-file --batch
trace git-lfs: tq: starting 3 transfer workers
trace git-lfs: tq: running as batched queue, batch size of 100
trace git-lfs: fetch MYFILE1 [10b3c7f98cc1871ef0c561faf3bbc825fc2ef6e5b5a616ed8d508b0536e45688]
trace git-lfs: fetch MYFILE2 [93181c18594396161dd77a27cfb89b5cd7bac59996d3b087ecb2755f8a8d8752]
trace git-lfs: fetch MYFILE3 [7d4b79538cfb737611217058ed9f6d61a33c5d0fc9dbdfb3994285930cde80d2]
trace git-lfs: fetch MYFILE4 [51668dfb7e3e6e5cd5429a5a185c1ca9a13be8711f6453c449fa37e4af79ff19]
trace git-lfs: fetch MYFILE5 [371f890023bb3a17f276dd7feadc2aaa5106002bf1b7078f20131d4d65bb2c5c]
trace git-lfs: tq: sending batch of size 5
trace git-lfs: api: batch 5 files
trace git-lfs: HTTP: POST https://tampe125@github.com/MYORG/MYREPO.git/info/lfs/objects/batch
trace git-lfs: HTTP: 401
trace git-lfs: setting repository access to basic
trace git-lfs: run_command: 'git' config lfs.https://tampe125@github.com/MYORG/MYREPO.git/info/lfs.access basic
trace git-lfs: api: batch not authorized, submitting with auth
```
Looking at the trace, it seems it's switching back and forth from basic and auth. I just cut the text, but that 401 error goes on endlessly

**Comment by technoweenie** [comment]
@tampe125: I think there's an issue with Git LFS for users that don't have git credentials setup. Your best bet is to use SSH or setup a git credential helper for now.

https://help.github.com/articles/caching-your-github-password-in-git/#platform-windows

**Comment by tampe125** [comment]
Just for the records, I wasn't able to use the `wincred` helper, so I switched to the `cache` one.
After doing that, everything seems working fine.
I hope the fix will come shortly

**Comment by technoweenie** [comment]
The `cache` helper only caches for 5 minutes. This [guide](https://help.github.com/articles/caching-your-github-password-in-git/#platform-linux) shows you how to increase the timeout from 5m.

I wasn't able to fix this issue in Windows for v1.1. However, Git LFS will halt if it asks for credentials but doesn't receive any. No more infinite auth loops.

**Comment by technoweenie** [comment]
Just FYI: I just replicated this on my mac. I think the progress output is pushing the username/password prompts all the way to the right. That's sometimes you can't see it.

```
Git LFS: (1 of 4 files) 16.64 KB / 818.94 KB                                                                                                                                                                                                                                                                                                                               Username for 'https://github-cloud.s3.amazonaws.com':
Git LFS: (1 of 4 files) 101.27 KB / 818.94 KB
```

Now that I know what causes it, we should be able to get a fix out in 1.1.1 :)
